modules/setting-up-cpu-manager.adoc

```diff
@@ -172,24 +179,50 @@ QoS Class:       Guaranteed
 Node-Selectors:  cpumanager=true
 ----
 
-. Verify that the `cgroups` are set up correctly. Get the process ID (PID) of the `pause` process:
+. Verify that a CPU has been exclusively assigned to the pod by running the following command:
++
+[source,terminal]
+----
+# oc describe node --selector='cpumanager=true' | grep -i cpumanager- -B2
+----
++
+.Example output
+[source,terminal]
+----
+NAMESPACE    NAME                CPU Requests  CPU Limits  Memory Requests  Memory Limits  Age
+cpuman       cpumanager-mlrrz    1 (28%)       1 (28%)     1G (13%)         1G (13%)       27m
+----
+
+. Verify that the `cgroups` are set up correctly. Get the process ID (PID) of the `pause` process by running the following command:
 +
 [source,terminal]
 ----
+# oc debug node/perf-node.example.com
+sh-4.2# systemctl status | grep -B5 pause
+----
++
+[NOTE]
+====
+If the output returns multiple pause process entries, you must identify the correct pause process.
+====
++
+.Example output
+[source,terminal]
+----
 # ├─init.scope
 │ └─1 /usr/lib/systemd/systemd --switched-root --system --deserialize 17
 └─kubepods.slice
   ├─kubepods-pod69c01f8e_6b74_11e9_ac0f_0a2b62178a22.slice
   │ ├─crio-b5437308f1a574c542bdf08563b865c0345c8f8c0b0a655612c.scope
   │ └─32706 /pause
 ----
-+
-Pods of quality of service (QoS) tier `Guaranteed` are placed within the `kubepods.slice`. Pods of other QoS tiers end up in child `cgroups` of `kubepods`:
+
+. Verify that Pods of quality of service (QoS) tier `Guaranteed` are placed within the `kubepods.slice`. Pods of other QoS tiers end up in child `cgroups` of `kubepods`:
```

Comment:
No need to capitalize "Pods" here if the document is referring to the general pod concept rather than an API object instance.

```suggestion
. Verify that pods of quality of service (QoS) tier `Guaranteed` are placed within the `kubepods.slice` [add noun here]. Pods of other QoS tiers end up in child `cgroups` [add noun here] of `kubepods` [add noun here]:
```